### PR TITLE
add gen_statem:call[2/3] to  eqwalizer_support

### PR DESCRIPTION
**Pull request by sg2342**

### CODE REVIEW

**Comment by ilya-klyuchnikov** [comment]
Thanks. - It turns out that we already have it in our "internal eqwalizer_specs" - I will upstream what we have tomorrow.

File: eqwalizer_support/src/eqwalizer_specs.erl [hunk 1]
```diff
@@ -312,6 +312,14 @@ when
 -spec 'gen_server:call'(gen_server:server_ref(), term(), timeout()) -> eqwalizer:dynamic().
 'gen_server:call'(_, _, _) -> error(eqwalizer_specs).
 
+%% -------- gen_statem --------
+
+-spec 'gen_statem:call'(gen_statem:server_ref(), term()) -> eqwalizer:dynamic().
+'gen_statem:call'(_, _) -> error(eqwalizer_specs).
+
+-spec 'gen_statem:call'(gen_statem:server_ref(), term(), timeout()) -> eqwalizer:dynamic().
```
Comment (ilya-klyuchnikov) [inline eqwalizer_support/src/eqwalizer_specs.erl:320]: It should be 

```erlang
-spec 'gen_statem:call'(gen_statem:server_ref(), term(), timeout() | {clean_timeout, timeout()} |  {dirty_timeout, timeout()} ) -> eqwalizer:dynamic().
```

**Comment by ilya-klyuchnikov** [comment]
Please fix the spec for `call/3` and it would be good to merge. Thanks!

File: eqwalizer_support/src/eqwalizer_specs.erl (hunk 1 above)
Comment (sg2342) [reply eqwalizer_support/src/eqwalizer_specs.erl]: fixed and force-pushed.

**Pull request closed without merging**